bakerydemo/static/css/main.css

```diff
@@ -1347,4 +1327,71 @@ input[type="radio"] {
     max-width: 180px;
     max-height: 180px;
   }
+}
+
+/* ---- Location Card ---- */
+
+.location-card {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto 50px;
+  text-align: left;
+}
+
+.location-card:hover .location-card__image img {
+  transform: scale(1.05);
+}
+
+.location-card:hover .location-card__title {
+  color: var(--dark-orange);
+  text-decoration: underline;
+}
+
+.location-card__image {
+  margin-bottom: 0;
+  overflow: hidden;
+}
+
+.location-card__image img {
```

Comment:
suggestion: add `object-fit: cover` so the images don't appear stretched

![bakery-object-fit](https://user-images.githubusercontent.com/31627284/166470729-9ed22f63-70ef-4d80-b361-e94acc25f790.gif)

Reply:
Thank you, that's much better